sphinx/testing/util.py

```diff
@@ -91,8 +91,8 @@ def __init__(
         status: IO | None = None,
         warning: IO | None = None,
         tags: list[str] | None = None,
-        docutils_conf: str | None = None,
-        parallel: int = 0,
+        docutilsconf: str | None = None,
```

Comment:
```suggestion
        docutils_conf: str | None = None,
```

This class can still be public API so we shouldn't change this name.

Reply:
I reverted this. It was a mistake.